core/storage/statistics/gae_models.py

```diff
@@ -781,7 +857,13 @@ def get_new_event_entity_id(cls, exp_id, user_id):
             user_id))
 
     @classmethod
-    def create(cls, exp_id, user_id, rating, old_rating):
+    def create(
+            cls,
+            exp_id: str,
+            user_id: str,
+            rating: int,
+            old_rating: Optional[str]
```

Comment:
Why is this a string? Seems weird and does not match with the docstring.

More generally, please do a check to ensure that your types and the docstrings are fully aligned in all the functions that you're touching in this (and other) PRs. Otherwise developers are going to be confused.

Reply:
My bad. It must be int as per the storage model definition. Changed.

Reply:
Please confirm whether you've checked all the other docstrings.

Reply:
Checked and found 3 places where None was missing from the docstring. Changed. Thanks!